packages/bundler-sdk-viem/test/populateBundle.test.ts

```diff
@@ -4773,6 +4773,93 @@ describe("populateBundle", () => {
           ); // we normally didn't experienced any slippage
         },
       );
+
+      test[ChainId.EthMainnet](
+        "should wrap ETH",
+        async ({ client, config }) => {
+          const assets = parseEther("1.234567");
+
+          await client.deal({
+            amount: assets,
+          });
+
+          const initialBalance = await client.balanceOf();
+
+          const block = await client.getBlock();
+
+          const { result } = await renderHook(config, () =>
+            useSimulationState({
+              marketIds: [],
+              users: [client.account.address, generalAdapter1],
+              tokens: [NATIVE_ADDRESS, wNative],
+              vaults: [],
+              block,
+            }),
+          );
+
+          await waitFor(() => expect(result.current.isFetchingAny).toBeFalsy());
+
+          const data = result.current.data!;
+
+          const { operations, bundle } = await setupTestBundle(
+            client,
+            data,
+            [
+              {
+                type: "Erc20_Wrap",
+                sender: client.account.address,
+                address: wNative,
+                args: {
+                  amount: assets,
+                  owner: client.account.address,
+                  slippage: DEFAULT_SLIPPAGE_TOLERANCE,
+                },
+              },
+            ],
+            { gasPrice: 0n },
```

Comment:
gasPrice should be 1 or at least deterministic so the gas fees should be too
But let's keep this